Makefile

```diff
@@ -33,4 +33,7 @@ purge: confirm ## Purge all containers and volumes
 	@$(DOCKER_COMPOSE) -f $(DOCKER_COMPOSE_FILE) down -v
 
 build: ## (re)Build all images or c=<name> containers
-	@$(DOCKER_COMPOSE) -f $(DOCKER_COMPOSE_FILE) build $(c)
\ No newline at end of file
+	@$(DOCKER_COMPOSE) -f $(DOCKER_COMPOSE_FILE) build $(c)
+
+migrations: ## Makemigrations django app
+	@$(DOCKER_COMPOSE) -f $(DOCKER_COMPOSE_FILE) exec app python manage.py makemigrations
```

Comment:
Nice! But I would add something more generic like:
```
manage/%: ## Execute manage commands
	@$(DOCKER_COMPOSE) -f $(DOCKER_COMPOSE_FILE) exec app python manage.py $*
```

Reply:
Ohh! You are right! Your solution is much better! thank @jbagot